core/qt-ble.cpp

```diff
@@ -247,9 +247,32 @@ BLEObject::~BLEObject()
 	delete controller;
 }
 
+/*
+ * The McLean Extreme has just one vendor service, but then inside that
+ * service it has several characteristics, and it's not obvious which
+ * ones are the read/write ones.
+ *
+ * So just make sure to skip the ones we don't want.
+ *
+ * The proper ones are:
+ *
+ *   Microchip service UUID:  49535343-fe7d-4ae5-8fa9-9fafd205e455
+ *            TX characteristic: 49535343-1e4d-4bd9-ba61-23c647249616
+ *            RX characteristic: 49535343-8841-43f4-a8d4-ecbe34729bb3
+ */
+static const struct uuid_match skip_characteristics[] = {
+	{ "49535343-6daa-4d02-abf6-19569aca69fe", "McLean Extreme Avoid" },
+	{ "49535343-aca3-481c-91ec-d85e28a60318", "McLean Extreme Avoid" },
+	{ "49535343-026e-3a9b-954c-97daef17e26e", "McLean Extreme Avoid" },
+	{ "49535343-4c8a-39b3-2f49-511cff073b7e", "McLean Extreme Avoid" },
+	{ NULL, }
+};
```

Comment:
I'm curious - why list the ones we don't want and not just list the ones that we want?

Reply:
I guess it makes the code below easier? It just seems like it would overall be more error prone?

Reply:
thinking more about how a positive flow would work, I think I like what you did better